src/sorcha_cmdline/run.py

```diff
@@ -116,6 +118,16 @@ def main():
         default=None,
     )
 
+    # outputting an extra error message to the prompt if no arguments are included.
+    # we found from beta testing that a user unfamilar with linux/unix may not
+    # know the -h flag is what they need to get help
+
+    if len(sys.argv) == 1:
+        print(
+            "\n\nError: No command line arguments provided!!! \nUse the -h flag to see descriptions of the required and optional arguments needed to run Sorcha \n",
+            file=sys.stderr,
```

Comment:
I'd propose to do something like this here:

```
parser.print_usage(file=sys.stderr)
print("For more detailed help run: sorcha run --help", file=sys.stderr)
```

A better soluton (which would solve this for all subcommends) is to derive a class from`ArgumentParser` and override the `print_usage` function. Example:

```
class SorchaArgumentParser(argparse.ArgumentParser):
    def print_usage(self, file=None):
        super().print_usage(file)
        print("For more detailed help run: sorcha run --help", file=file)
```

and then just change line 9 to `parser = SorchaArgumentParser`.

(you'd probably want to put SorchaArgumentParser in some util module in `sorcha_cmdline` package).

Reply:
Correction, you'll need something like this:
```
class SorchaArgumentParser(argparse.ArgumentParser):
    def __init__(self, cmdname, *args, **kwargs):
        self.cmdname = cmdname
        return super().__init__(**args, **kwargs)

    def print_usage(self, file=None):
        super().print_usage(file)
        print(f"For more detailed help run: sorcha {self.cmdname} --help", file=file)
```
so as not to hardcode the subcommand name.